cmake/alpakaCommon.cmake

```diff
@@ -415,11 +415,11 @@ if(ALPAKA_ACC_GPU_CUDA_ENABLE)
                 endif()
             endif()
 
-            # NOTE: Since CUDA 10.2 this option is also alternatively called '--extended-lambda'
+            # NOTE: Before CUDA 10.2 this option was also alternatively called '--expt-extended-lambda'
```

Comment:
I don't think this note is still necessary.

Reply:
ok, dropped.